src/curve_fitting/canvas.cljs

```diff
@@ -42,7 +88,8 @@
                                :max-y  10
                                :units-per-tick 1})
         ctx (.getContext (:canvas graph) "2d")]
-
+    (draw-x-axis graph)
+    (draw-y-axis graph)
```

Comment:
Since we repaint the graph every time it's not yet clear to me why we need to do different things during the initial render vs later renders. 🤔